### PR TITLE
1604 additional review page fixes

### PR DESCRIPTION
**Pull request by guatemartin**

### CODE REVIEW

File: src/containers/Review/ReviewWrapper.tsx [hunk 1]
```diff
@@ -54,7 +54,11 @@ const ReviewWrapper: React.FC<ReviewWrapperProps> = ({ structure }) => {
   } = fullStructure
 
   const getTabFromQuery = (tabQuery: string | undefined) => {
-    const index = tabIdentifiers.findIndex((tabName) => tabName === tabQuery)
+    // If mobile, we reverse array to improve "wrapping" of tab elements.
+    // So reverse array to make 'overview' the default tab
+    const index = isMobile
+      ? tabIdentifiers.reverse().findIndex((tabName) => tabName === tabQuery)
+      : tabIdentifiers.findIndex((tabName) => tabName === tabQuery)
```
Comment (guatemartin) [inline src/containers/Review/ReviewWrapper.tsx:61]: Additional change.  In mobile array was reversed, so 'Documents' was showing as the default tab.  This makes 'Overview' the default tab again in Mobile.

File: src/containers/Review/ReviewWrapper.tsx [hunk 2]
```diff
@@ -155,7 +156,9 @@ const ReviewHomeHeader: React.FC<ReviewHomeProps> = ({
   const [prevQueryString] = useState(location?.state?.prevQuery)
 
   if (!tab) {
-    updateQuery({ tab: tabIdentifiers[0] })
+    !isMobile
+      ? updateQuery({ tab: tabIdentifiers[0] })
+      : updateQuery({ tab: tabIdentifiers[tabIdentifiers.length - 1] })
```
Comment (guatemartin) [inline src/containers/Review/ReviewWrapper.tsx:161]: This makes 'overview' the default tab again in mobile as we are reversing the tab panes in mobile.

File: semantic/src/site/globals/site.overrides [hunk 3]
```diff
@@ -617,6 +617,8 @@ a:hover {
   color: @headersInteractiveMed;
   font-size: @fontSmall;
   font-weight: @semiboldWeight;
+  flex-wrap: wrap;
+  gap: 10px;
 }
```
Comment (guatemartin) [inline semantic/src/site/globals/site.overrides:622]: wraps Stage Label and gives it a gap without affecting desktop.  'flex : 1' in label.overrides also allows the dropdown to occupy more of the space and not wrap the text inside the dropdown.
assTabFixLabel

File: src/containers/Review/overview/Overview.tsx [hunk 4]
```diff
@@ -92,7 +92,10 @@ export const Overview: React.FC<{
                 // EXPIRED, so cancelled deadlines don't cause the button to show.
                 ((applicantDeadline.isActive && outcome === ApplicationOutcome.Pending) ||
                   (!applicantDeadline.isActive && outcome === ApplicationOutcome.Expired)) && (
-                  <div className="flex-row-start-center" style={{ gap: 10, marginTop: 30 }}>
+                  <div
+                    className="flex-row-start-center wrap"
+                    style={{ gap: 15, marginTop: isMobile ? 15 : 30 }}
```
Comment (guatemartin) [inline src/containers/Review/overview/Overview.tsx:97]: This is a little awkward looking, but wrapping the button seemed the best option.
Was also trying to fix no having gaps btn labels (red circles) when wrapping... but will leave for another time.

overview_no_gaps

File: semantic/src/site/globals/site.overrides [hunk 5]
```diff
@@ -2841,9 +2843,15 @@ h6:first-child {
...
+  #notes-tab {
+    padding: 5px 0px 0px 5px;
+    .options-row {
+      flex-direction: column;
+      align-items: flex-start;      
+    }
+    .item-container {
+      width: 100%;
+    }
   }
```
Comment (guatemartin) [inline semantic/src/site/globals/site.overrides:2855]: class names get confusing in this area, but hopefully this aligns and spaces things out better in notes tab

notes_tab_spacing_fix

File: semantic/src/site/globals/site.overrides [hunk 6]
```diff
@@ -2818,6 +2820,7 @@ h6:first-child {
...
   #overview .left-item {
     min-width: unset;
+    padding-right: 5px;
```
Comment (guatemartin) [inline semantic/src/site/globals/site.overrides:2823]: Fix small gap in labels as they wrap.

label_gap_finish

File: src/containers/Review/ReviewWrapper.tsx [hunk 7]
```diff
@@ -107,11 +107,7 @@ const ReviewWrapper: React.FC<ReviewWrapperProps> = ({ structure }) => {
             orgName={org?.name as string}
           />
           <div id="review-home-content">
-            {isMobile && structure.stages.length <= 2 ? (
-              <ReviewProgress structure={structure} />
-            ) : (
-              !isMobile && <ReviewProgress structure={structure} />
-            )}
+            {!isMobile && <ReviewProgress structure={structure} />}
```
Comment (guatemartin) [inline src/containers/Review/ReviewWrapper.tsx:110]: progress bar completely on mobile.

File: src/containers/Review/assignment/ReviewSectionRow.tsx [hunk 8]
```diff
@@ -88,18 +88,10 @@ const ReviewSectionRow: React.FC<ReviewSectionRowProps> = ({
     <div key={`section_${sectionCode}_assignment_${assignmentId}`}>
       {canRenderRow && (
         <Grid stackable columns={4} verticalAlign="middle">
-          <Grid.Column className="custom-assignment-grid-column">
```
Comment (guatemartin) [inline src/containers/Review/assignment/ReviewSectionRow.tsx:91]: I think I added this in an earlier PR.  I hadn't realized that the components in the grid.col where actually returning grid.columns themselves. So removed them.

File: src/containers/Review/overview/Overview.tsx (hunk 4 above)
Comment (CarlosNZ) [reply src/containers/Review/overview/Overview.tsx:97]: Yeah, that's good thanks. I've noticed that when you use `gap:` on a flex block, the gap gets applied vertically as well when they wrap. Which is kind of annoying (I've been caught out by it a several times), but not really worth the hassle to try and fix. This is great, thanks.

**Review by CarlosNZ — APPROVED** [review]
That's great, this area is all looking pretty good now!

There's just one little issue with the tab ordering (again!), which I've detailed in a comment. But see what you can do with that and then merge this whenever you're ready. (I'll be out for the rest of the day today)

**Comment by CarlosNZ** [comment]
I'll merge this now @guatemartin 

Cheers.

**Pull request merged**